
<html>
    <head>
        <style>div#main {width: 90%; margin: auto}</style>
        <title>TeraClientServer</title>
                     <meta charset="utf-8" />             <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />             <meta name="description" content="" />             <meta name="author" content="kent chen" />             <meta name="viewport" content="width=device-width; initial-scale=1.0" />             <link href="http://netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.min.css" rel="stylesheet">             <script src="http://netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/js/bootstrap.min.js"></script>
    </htad>
    <body>
        <div id="main">
            <h1>TeraClientServer</h1>
            <hr>
            <h3>Summary</h3>
                             <table class="table table-hover table-bordered" style="width:200px">                 <tbody>                 <tr class="info"><td>Ran Cases</td><td><strong>63</strong></td></tr>                 <tr class="success"><td>Passed</td><td><strong>0</strong></td></tr>                 <tr class="error"><td>Failed</td><td><strong>63</strong></td></tr>                 </tbody>                 </table>
            <hr>
            <h3>Detail</h3>
                         <table class="table table-hover">             <thead><tr><th>Case ID</th><th>Test Result</th><th>Log Message</th><th>Duration Time</th></tr></thead> <tr><td>TeraClientServer_00000</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_01002</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_01003</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_01004</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_01005</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_01006</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_01007</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_01008</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_01009</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_01010</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_01011</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_01012</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_01013</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02001</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02002</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02003</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02004</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02005</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02006</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02007</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02008</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02009</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02010</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02011</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02012</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02013</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02014</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02015</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02016</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02017</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02018</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02019</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02020</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02021</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02022</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02023</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02024</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02025</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02026</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02027</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02028</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02029</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02030</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02031</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02032</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02033</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02034</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_02035</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_03001</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_03002</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_03003</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_05001</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_05002</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_05003</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_05004</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_05005</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_05006</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_05007</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_05008</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_05009</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_05010</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr><tr><td>TeraClientServer_99999</td><td style="background-color:#D0E9C6">Error</td><td>Traceback (most recent call last):
  File "/home/tester/Documents/pi-tester/Engine/TestEngine.py", line 87, in _invoke_test_case
    mod = __import__('TestCases.{0}.{0}'.format(case_classify), fromlist=[case_id])
  File "/home/tester/Documents/pi-tester/TestCases/TeraClientServer/TeraClientServer.py", line 8, in <module>
    import jwt
ImportError: No module named jwt
</td><td>0</td></tr>             </table>
        </div>
    </body>
</html>
        